lib/Service/ObjectService.php

```diff
@@ -232,7 +232,7 @@ public function saveObject(int $register, int $schema, array $object): ObjectEnt
 			$object['id'] = $objectEntity->getUuid();
 		}
 
-		$oldObject = $objectEntity->getObject();
+		$oldObject = clone $objectEntity;
 		$objectEntity->setObject($object);
```

Comment:
Please help me understand what is happening here because I don't understand why we would clone $objectEntity

Reply:
This I can help with. With `clone` you create a hard copy of the object. That means that it is memory separated. That is necessary to create te correct audit-log. By just running `$oldObject = $objectEntity`, changes on `$objectEntity` would also change `$oldObject`, while now, `$oldObject` remains unchanged while `$objectEntity` changes.